README.md

```diff
@@ -1,5 +1,9 @@
 [![CircleCI](https://circleci.com/gh/dof-dss/nicsdru_nidirect_theme.svg?style=svg)](https://circleci.com/gh/dof-dss/nicsdru_nidirect_theme)
 
+# DEPRECIATED 
```

Comment:
Good to have. Should this be 'deprecated' (going towards obsolescence) rather than 'depreciated' (to decrease in value)?

Reply:
🤦